doc/source/examples/unresolved-cfd-dem/single-particle-sedimentation/single-particle-sedimentation.rst

```diff
@@ -0,0 +1,362 @@
+
+----------------------------------
+Features
+----------------------------------
+
+- Solvers: ``lethe-particles`` and ``lethe-fluid-particles``
+- Three-dimensional problem
+- Displays the selection of models and physical properties
+- Simulates a single particle sedimentation in water
+- Uses the ``qcm`` void fraction scheme with ``gauss-lobatto`` quadrature
+- Python post-processing script using `PyVista <https://docs.pyvista.org/>`_
+
+
+---------------------------
+Files Used in This Example
+---------------------------
+
+All files mentioned below are located in the example's folder (``examples/unresolved-cfd-dem/single-particle-sedimentation``).
+
+- Parameter file for initial particles generation: ``initial-particles.prm``
+- Parameter file for CFD-DEM simulation of the Boycott effect: ``single-particle-sedimentation.prm``
+- Post-processing script: ``single_particle_sedimentation.py``
+- Module for post-processing script: ``single_particle_sedimentation_funcs.py``
+
+
+-----------------------
+Description of the Case
+-----------------------
+
+This example simulates the sedimentation of a single particle in a water tank due to gravity. As all unresolved CFD-DEM applications and examples, we start by introducing the particle in our domain by running a DEM simulation (using ``lethe-particles``). The DEM simulation generates checkpoint files that will be used by our unresolved CFD-DEM application (``lethe-fluid-particles``) to determine the initial state (position, velocity, etc.) of the particles.
+
+This is most probably the simplest application case of CFD-DEM we can think of and hence, it is very useful to learn the fundamentals of the method. In Lethe, we use examples such as this to guide the user on how to launch a simple example and to track the performance of our code.
+
+We use this case to assess the grid convergence of the `quadrature centered method (QCM) <../../../parameters/unresolved-cfd-dem/void-fraction.html>`_, by Geitani and Blais [#geitani2023]_. We check if the adequate void fraction is recovered and we monitor the terminal velocity of the sedimenting particle. We use the properties given by Ferreira et al. [#ferreira2023]_ for the particle and fluid.
+
+-------------------
+DEM Parameter File
+-------------------
+
+The DEM subsections are as follows.
+
+Mesh
+~~~~~
+
+In this example, we are simulating a rectangular-based tank. We use the deal.II GridGenerator to generate a hyper rectangle that is subdivided along its height:
+
+.. code-block:: text
+
+    subsection mesh
+      set type           = dealii
+      set grid type      = subdivided_hyper_rectangle
+      set grid arguments = 20,40,20:-0.02663,0,-0.02663:0.02663,0.10652,0.02663:true
+    end
```

Comment:
```suggestion
    subsection mesh
      set type           = dealii
      set grid type      = subdivided_hyper_rectangle
      set grid arguments = 20,40,20:-0.02663,0,-0.02663:0.02663,0.10652,0.02663:false
    end
```

No need to colorize, you have slip everywhere, so just put false instead of true for colorize to make things simpler